src/tests/functional/visit_tests.ts

```diff
@@ -98,6 +98,21 @@ export class VisitTests extends TurboDriveTestCase {
     this.assert.isTrue(fetchResponseResult.responseHTML.indexOf('An element with an ID') > -1)
   }
 
+  async "test cache does not override response after redirect"() {
+    await this.remote.execute(() => {
+      const cachedElement = document.createElement("some-cached-element")
+      document.body.appendChild(cachedElement)
```

Comment:
Is it significant that this is an appended element? Could an `<input>` with text typed into it suit this test as state to preserve?

Reply:
In my experience, the browser stores `<input>` text in memory, not in the DOM.  Since turbo only caches DOM, testing cache through text input would not work.  

Alternatively, I could add (or remove) a attribute to an existing DOM item (e.g. `data-cached="true"`)

Reply:
@seanpdoyle - thumbs up means you are good with the explanation or you want an update to the test with one of the suggestions?

Reply:
The current version works. Thank you for uncovering and digging into this!